gpu-operator/gpu-operator-mig.rst

```diff
@@ -323,6 +323,41 @@ The following steps show how to update a GPU on a node to the ``3g.40gb`` profil
       }
 
 
+Example: Custom MIG Configuration During Installation
+=====================================================
+
+By default, the Operator creates the ``default-mig-parted-config`` config map and MIG Manager is configured to read profiles from that config map.
+
+You can use the ``values.yaml`` file when you install or upgrade the Operator to create a config map with a custom configuration.
+
+#. In your ``values.yaml`` file, add the data for the config map, like the following example:
+
+   .. literalinclude:: manifests/input/mig-cm-values.yaml
+      :language: yaml
+
+#. If the custom configuration specifies more than one instance profile, set the strategy to ``mixed``:
+
+   .. code-block:: console
+
+      $ kubectl patch clusterpolicies.nvidia.com/cluster-policy \
+          --type='json' \
+          -p='[{"op":"replace", "path":"/spec/mig/strategy", "value":"mixed"}]'
+
+#. Patch the cluster policy so MIG Manager uses the custom config map:
+
+   .. code-block:: console
+
+      $ kubectl patch clusterpolicies.nvidia.com/cluster-policy \
+          --type='json' \
+          -p='[{"op":"replace", "path":"/spec/migManager/config/name", "value":"custom-mig-config"}]'
```

Comment:
This step is not needed as `migManager.config.name` is already set to `custom-mig-config` in values.yaml.